nifi-stateless/nifi-stateless-bootstrap/src/main/java/org/apache/nifi/stateless/NiFiStateless.java

```diff
@@ -72,14 +73,6 @@ public static void main(final String[] args) throws IOException, ClassNotFoundEx
             for (final File narFile : narFiles) {
                  NarUnpacker.unpackNar(narFile, narWorkingDirectory);
```

Comment:
@SamHjelmfelt i think you want this to be a directory under narworkingdirectory.  traditional nifi writes to ./work/nar/...    you might want to do ./work/stateless-nar.  They should avoid sharing as they can have different running cycles and nifi has hotloading where as stateless presumably would not/etc..